bindings/rust/src/lib.rs

```diff
@@ -8,11 +8,35 @@
 
 mod sys;
 
+use std::ptr::NonNull;
+
 /// Parse and validate the input according to WebAssembly 1.0 rules. Returns true if the supplied input is valid.
 pub fn validate<T: AsRef<[u8]>>(input: T) -> bool {
     unsafe { sys::fizzy_validate(input.as_ref().as_ptr(), input.as_ref().len()) }
 }
 
+/// A parsed and validated WebAssembly 1.0 module.
```

Comment:
I don't think it make sense to mention 1.0 anywhere in the comments.

Reply:
We use the "1.0" in the C API too.

Reply:
I think this is a mistake. We should not randomly in many places in comments information what version of WebAssembly is supported.

Reply:
I suggest we clean up documentation at once (Rust, C, C++), if we want to remove this "1.0" stuff.

Reply:
This is not my preference, but ok.